tests/test_modifiers.py

```diff
@@ -1,10 +1,10 @@
 import pytest
 
-from server.modifiers import FOUR_SPACES, STATEMENTS, Modifiers
+from server.modifiers import FOUR_SPACES, STATEMENTS, TYPES, Modifiers
 
 test_input = [
     "def say_hello() -> str:\n",
-    '    return "Hello!"\n',
+    '    if 1 == True and 0 == False: return "Hello!"\n',
```

Comment:
Not a big issue but make sure this would be correct. Add the `return "Hello!"\n` to the next line and add `\n` at the end of the statement

Reply:
Isn't one line "if" statement correct tho? I've done it this way to avoid changing test_removing_indentation

Reply:
Syntactically correct, yes but not stylistically correct. I see what you mean with the removing_indentation. Could you perhaps create a fixture instead specific to the reversing_booleans?

Reply:
Done